pyteal/ast/subroutine.py

```diff
@@ -230,7 +253,9 @@ def invoke(
                         f"should have ABI typespec {arg_type} but got {arg.type_spec()}"
                     )
 
-        return SubroutineCall(self, args)
+        return SubroutineCall(
+            self, args, output_kwarg=_OutputKwArgInfo.from_dict(self.output_kwarg)
```

Comment:
@ahangsu I opened an optional PR for your review based on local changes I made while reviewing `output_kwarg`.  I'd appreciate you taking a look as time allows - https://github.com/algorand/pyteal/pull/311.

Treat as optional and feel welcomed to keep as is.

Reply:
Resolving - We agreed to close the child PR.